examples/postgres/example_flyway_postgres_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	tcnetwork "github.com/testcontainers/testcontainers-go/network"
 )
 
-func ExampleRunContainer() {
+func ExampleFlyway_postgres() {
```

Comment:
maybe uppercase for `postgres` ? looks more consistent. Personal preference though

Reply:
Thanks good to know @mdelapenya